scripts/tutorials/wdl/README.md

```diff
@@ -0,0 +1,31 @@
+### Easily find information in each tutorial
+---
+
+The WDL scripts in this folder are designed to teach you aspects and features of WDL, using GATK as it's main tool. They follow the tutorial documents located on the WDL website, under the Tutorials category
+
+- https://software.broadinstitute.org/wdl/userguide/topic?name=wdl-tutorials
+
+Each  tutorial script located in this folder will have a link in its header that points to its specific tutorial document. You can copy and paste that link into your browser window and follow along with the document in order to write your own script. These are designed in a way that you should be using the scripts uploaded to this folder as a means of checking your work. 
+
+### Example data
+---
+
+Each tutorial document will link you to the specific `.zip` file associated with each script. These `.zip` files contain all the example data you will need to run the script, in addition to the sample script itself. You can find these `.zip` files here:
+
+- https://drive.google.com/drive/u/0/folders/0BwTg3aXzGxEDMUs0cURPcnFiQzA
+
+### Be sure to generate a JSON for your setup
```

Comment:
How about using dummy paths like `/path/to/foo.bam`? They're widely used in software documentation. And they would at least show users what the full json looks like when the values have all been filled in. 

Ultimately I'm thinking of making a WDL cloud bucket with all the example files and docker urls that would work out of the box if run on eg FireCloud. That's not going to be ready in time for this but in the meantime I feel like providing dummy paths is the lesser of two evils (the greater being not having an example json at all).